stm32-modules/thermocycler-gen2/firmware/thermal/ads1115.cpp

```diff
@@ -102,7 +102,8 @@ auto ADC::read(uint16_t pin) -> ADC::ReadVal {
 
     static_cast<void>(release_lock());
     if ((!i2c_ret) || (notification_val == 0)) {
```

Comment:
[nit] this is pretty annoying to read, i think a series of ifs would be nicer, like
```
if (!i2c_ret) {
    return ReadVal(Error::ADCTimeout);
}
if (notification_val == 0) {
    return ReadVal(Error::I2CTimeout);
}
```